internal/utils/tenant/postgrest.go

```diff
@@ -0,0 +1,38 @@
+package tenant
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/supabase/cli/internal/utils"
+)
+
+var errPostgrestVersion = errors.New("PostgREST version not found.")
+
+type SwaggerInfo struct {
+	Title       string `json:"title"`
+	Description string `json:"description"`
+	Version     string `json:"version"`
+}
+
+type SwaggerResponse struct {
+	Swagger string      `json:"swagger"`
+	Info    SwaggerInfo `json:"info"`
+}
+
+func GetPostgrestVersion(ctx context.Context, projectRef string) (string, error) {
```

Comment:
Have you tested this? The PostgREST version in the OpenAPI can't be used directly as Docker image versions - it looks like `11.2.0 (c820efb)`

Reply:
Is it causing problems for you? I prefixed with `v` on line 37

The linked project I tested was on 11.1.0 though

Reply:
Yeah, just created a new project and the `.info.version` is the one above

Reply:
> I prefixed with v

The problem is the commit hash - the whitespace and everything after needs to be omitted

Reply:
Updated code to handle both cases.